ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -495,7 +495,7 @@ function setupSpellNotes(spellDomain) {
     });
   });
 
-  on('clicked:closepopoverspellnotes', closeAllSpellNotes);
+  on(`clicked:repeating_${spellDomain}:closepopoverspellnotes`, closeAllSpellNotes);
```

Comment:
I think we had a hidden bug in our code. I suspect that Roll20 tightened up their code, which broke this incorrect listener.
It should have had a repeating selector.

Reply:
How did you find that so fast?

Reply:
I ended up testing several button combinations, and eventually found this was missing.
A bit of trial and error, really.